### ISSUE TITLE
Confusing docs around webpack-cli, webpack-dev-server and CommonsChunkPlugin

### ISSUE THREAD

**Issue by kasperpeulen** [webpack/webpack.js.org#2286]
I'm reading the docs of webpack 4. But I find it confusing.

Here webpack-cli is recommanded:
https://webpack.js.org/guides/getting-started/

Here it is says to use webpack-command:
https://medium.com/webpack/webpacks-new-cli-hotness-d6a2063168a8

In the docs webpack-dev-server is recommended:
https://webpack.js.org/guides/development/#choosing-a-development-tool

But on the webpack-dev-server readme it recommends webpack-serve:
https://github.com/webpack/webpack-dev-server

Here the CommonsChunkPlugin is recommended to extract webpack's boilerplate and manifest:
https://webpack.js.org/guides/caching/#extracting-boilerplate

But on the CommonsChunkPlugin page itself, it says that it is deprecated:
https://webpack.js.org/plugins/commons-chunk-plugin/

**Comment by EugeneHlushko** [comment]
This is solely IMO but i suggest to follow current version of the docs regarding the bleeding edge suggestion to use `webpack-command`, as soon as `webpack-command` is settled and documented in the guides you would make the switch for your current projects.

CommonsChunkPlugin is no longer recommended by the guide, there was a pr that was merged yesterday. Please visit the page again to see the latest version https://webpack.js.org/guides/caching/#extracting-boilerplate

To summarize, the docs are source of truth unless you are researching whats new and playing around with tooling. And issues that we had on the guide page, are resolvable via PRs and they are more than welcome! 

Hope this helps

**Comment by kasperpeulen** [comment]
Thanks @EugeneHlushko.

**Comment by EugeneHlushko** [comment]
@kasperpeulen glad i could help, cheers
